rga: add page (#7818)
tldr-pages/tldr: pages/common/rga.md

# rga

> Ripgrep wrapper with rich file type searching capabilities.
> More information: <https://github.com/phiresky/ripgrep-all>.

> - Recursively search the current directory for a regular expression:

Author: How about:
```suggestion
- Recursively search for a pattern in all files in the current directory:
```

> - Recursively search the current directory for a regular expression:

Reviewer: We use to start with a verb, but yes it can works.

> - Recursively search the current directory for a regular expression:

Author: ```suggestion
- Search recursively for a pattern in all files in the current directory:
```
Maybe this?

> - Recursively search the current directory for a regular expression:

Reviewer: Yeah

`rga {{regular_expression}}`

> - List all known adapters:

Reviewer: I think this sounds better
```suggestion
- List available adapters:
```

`rga --rga-list-adapters`

- Change which adapters to use (e.g. ffmpeg, pandoc, poppler etc.):

> `rga --rga-adapters={{adapter,adapter}} {{regular_expression}}`

Reviewer 2: How about:
```suggestion
`rga --rga-adapters={{adapter1,adapter2}} {{regular_expression}}`
```

> - Use more accurate but slower matching by mime type:

Reviewer: ```suggestion
- Search for a pattern using the mime type instead of the file extension (slower):
```

`rga --rga-accurate {{regular_expression}}`

> - Show detailed help:

Reviewer: We use this usually
```suggestion
- Display help:
```

Author: There are two different `help` options on `rga`. According to the `-h`elp:
```
-h shows a concise overview
--help shows more detail and advanced options
```

What about:
```suggestion
- Display detailed help:
```

Author: Or maybe
```suggestion
- Display long help:
```
